tests/Functional/Types/StringBindingTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Types;
```

Comment:
This test is no longer about the data type, it's about the `ParameterType`. Might make sense to rename to `\Doctrine\DBAL\Tests\Functional\ParameterTypes\AsciiStringTest.php`.